package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.37.0",
+  "version": "1.37.1",
```

Comment:
Can you update version to `1.37.1-0.0.1` when you have a moment?
Current version is 1.37.1

Reply:
Done